templates/configmaps_st2-conf.yaml

```diff
@@ -22,15 +22,19 @@ data:
     api_url = http://{{ .Release.Name }}-st2api{{ template "enterpriseSuffix" . }}:9101/
     [coordination]
     url = etcd://{{ .Release.Name }}-etcd:2379
+    {{- if index .Values "rabbitmq-ha" "enabled" }}
     [messaging]
+    {{- end }}
     url = amqp://{{ required "rabbitmq-ha.rabbitmqUsername is required!" (index .Values "rabbitmq-ha" "rabbitmqUsername") }}:{{ required "rabbitmq-ha.rabbitmqPassword is required!" (index .Values "rabbitmq-ha" "rabbitmqPassword") }}@{{ .Release.Name }}-rabbitmq-ha-discovery:5672
```

Comment:
The condition block above should also include this RabbitMQ `url` string

Reply:
oops